File: dotnet/Core/Models/TodoEntity.cs
namespace Todo.Core.Models;

public class TodoEntity
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public DateTime DeadlineAt { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public bool IsDone { get; set; }
}

File: java/src/main/java/com/example/todo/entity/TodoEntity.java
package com.example.todo.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Date;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class TodoEntity {
    private int id;

    private Date createdAt;

    private Date updatedAt;

    private Date deadlineAt;

    private String title;

    private String description;

    private boolean isDone;
}
